### PR TITLE
#17 readability metrics first pass

### PR DESCRIPTION
**Pull request by ats0stv**
Added calculator for calculating the following readability metrics
* Flesch Reading Ease Score
* Flesch-Kincaid Grade Level
* Gunning Fog Index
* Automated Readability Index
* Simple Measure of Gobbledygook
* Coleman-Liau Index
* Linsear Write Score

The following is not implemented.
* Fry Readability Formula

**Defined the output model as follows.** 
```python
class OutputDataModel:
    def __init__(self):
        LOG.debug(f"{__name__} init")
        self.coreMetrics = CoreMetrics()
        self.readabilityMetrics = ReadabilityMetrics()
        self.category = ""
        self.description = ""
```

**The output object can be accessed as follows.**
```python
from lisibilite.Lisibilite import Lisibilite
    def sample():
        readabilityWithFile = Lisibilite("./resources/sample_text.txt")
        output = readabilityWithFile.getReadabilityMetrics()
        print(output)
```

**Sample Output**
```
*** Core Metrics
Total Sentences = 11
Total Words = 54
Total Hard Words = 51
Total Easy Words = 3
Total Syllables = 82
Total Characters = 260

*** Readability Metrics
Automated Readability Index  = 3.7023232323232307
Coleman-Liau Index  = 6.481481481481481
Flesch-Kincaid Grade Level  = 4.243063973063972
Flesch Reading Ease Score  = 73.38560606060608
Fry Readability Formula  = 0.0
Gunning Fog Index  = 39.74141414141414
Linsear Write Score  = 6.09090909090909
Simple Measure of Gobbledygook  = 15.429909175157395
Category = 
Description = 
```

### CODE REVIEW

File: code/lisibilite/Lisibilite.py [hunk 1]
```diff
@@ -2,8 +2,12 @@
 
 from utils.io.FileReader import FileReader
```
Comment (chhabriv) [inline code/lisibilite/Lisibilite.py:3]: I had added an import organiser - isort -> looked decent

File: code/lisibilite/Lisibilite.py [hunk 2]
```diff
@@ -19,27 +23,22 @@ def __init__(self, filename: str = None, contents: str = None):
 
         try:
             if filename:
-                self.metrics = self.__computeMetricsFromFile(filename)
-                self.computed = True
```
Comment (chhabriv) [inline code/lisibilite/Lisibilite.py:23]: Not needed anymore? Could remove from `init` also then?

File: code/lisibilite/Lisibilite.py [hunk 3]
```diff
@@ -19,27 +23,22 @@ def __init__(self, filename: str = None, contents: str = None):
...
-        Private: Method to compute the metrics from an input string
-        :param contents: The string for which the metrics should be computed
-        :return: Metrics object
+        The method to retrieve the Output Model
+        :return: The output model
         """
-        LOG.debug("Computing metrics from string")
-        metrics = CoreMetrics()
-        return metrics
+        return self.outputModel
```
Comment (chhabriv) [inline code/lisibilite/Lisibilite.py:41]: Should this be initialized to `None` in `init` ?
Or can use the computed flag is needed?

File: code/lisibilite/Lisibilite.py (hunk 1 above)
Comment (ats0stv) [reply code/lisibilite/Lisibilite.py:3]: Done

File: code/lisibilite/Lisibilite.py (hunk 3 above)
Comment (ats0stv) [reply code/lisibilite/Lisibilite.py:41]: Done

**Pull request merged**